src/PluginSuppression.php

```diff
@@ -60,11 +59,13 @@ public function register() {
 		// but there is no similar need to suppress the registration of Customizer controls in Transitional mode since
 		// there is no separate Customizer for AMP in Transitional mode (or legacy Reader mode).
 		if ( $this->is_reader_theme_request() ) {
-			add_action( 'plugins_loaded', [ $this, 'suppress_plugins' ], $priority );
+			$this->suppress_plugins();
 		} else {
+			$min_priority = defined( 'PHP_INT_MIN' ) ? PHP_INT_MIN : ~PHP_INT_MAX; // phpcs:ignore PHPCompatibility.Constants.NewConstants.php_int_minFound
+
 			// In Standard mode we _have_ to wait for the wp action because with the absence of a query parameter
 			// we have to rely on is_amp_endpoint() and the WP_Query to determine whether a plugin should be suppressed.
-			add_action( 'wp', [ $this, 'maybe_suppress_plugins' ], $priority );
+			add_action( 'wp', [ $this, 'maybe_suppress_plugins' ], $min_priority );
```

Comment:
Shouldn't this be `1` later?
```suggestion
			add_action( 'wp', [ $this, 'maybe_suppress_plugins' ], $min_priority + 1 );
```

Reply:
No, as this is for the `wp` action not the `plugins_loaded` action. So there's no conflict with `amp_bootstrap_plugin()` running at `plugins_loaded`.

Reply:
Yup... 🤦